### PR TITLE
Upstream ads removed, upstream integration options disabled by default

### PR DESCRIPTION
**Pull request by pboguslawski**
## Proposed change

Upstream ads removed, upstream integration options disabled by default.

## Type of change
- '1 - 🐞 bug 🐞'            - Bugfix (non-breaking change which fixes an issue)

## Additional information

- Related: 42374d0
- Author-Change-Id: IB#1046362

## Checklist

- [x] The code change is tested and works locally.(❗)
- [x] There is no commented out code in this PR.(❕)
- [ ] You improved or added new unit tests.(❕)
- [x] Local ZnunyCodePolicy run passes successfully.(❕)
- [x] Local unit tests pass.(❕)
- [ ] GitHub workflow ZnunyCodePolicy passes.(❗)
- [ ] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

**Comment by rkaldung** [comment]
Hi @pboguslawski,

Please follow the [contributing guidelines](https://github.com/znuny/Znuny/blob/rel-6_0-dev/CONTRIBUTING.md) and use the pull request template. And also, please point out if this is a bug fix or a feature.

**Comment by pboguslawski** [comment]
Done.

**Comment by dennykorsukewitz** [comment]
Hi @pboguslawski 

we have changed our rules regarding copyright.
We therefore ask you to remove your newly added copyrights.

```
Znuny is an open fork of an existing software.
So we have to respect the already given copyright of the original creators.

New files will be licensed using the AGPL Version 3. 
If you contribute code to the Znuny project you will get mentioned in the pull request incl. the commit, 
in CHANGES.md and in AUTHORS.md. 
We will not mention you in the file you provided or changed. 
Your work is highly appreciated and acknowledged but you contribute it to the project 
and your copyright will pass on to the fork itself.
```
https://github.com/znuny/Znuny/blob/rel-6_1-dev/CONTRIBUTING.md#licensing-copyright-and-credits

**Comment by pboguslawski** [comment]
Done.

File: Kernel/Config/Files/XML/Daemon.xml [hunk 1]
```diff
@@ -738,7 +738,7 @@
             </Hash>
         </Value>
     </Setting>
-    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="1" Valid="1" ReadOnly="1" ConfigLevel="100">
+    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="0" Valid="0" ConfigLevel="100">
```
Comment (dennykorsukewitz) [inline Kernel/Config/Files/XML/Daemon.xml:741]:
```suggestion
    <Setting Name="Daemon::SchedulerCronTaskManager::Task###SupportDataCollectAsynchronous" Required="1" Valid="1" ConfigLevel="100">
```

We will not disable this SysConfig by default.
However, we will remove `ReadOnly`.

File: Kernel/Config/Files/XML/Framework.xml [hunk 2]
```diff
@@ -7375,54 +7375,6 @@ via the Preferences button after logging in.
...
-                        <Item Key="Name" Translatable="1">Statistics</Item>
-                        <Item Key="Link">Action=AgentStatistics;Subaction=Overview</Item>
-                        <Item Key="LinkOption"></Item>
-                        <Item Key="NavBar">Reports</Item>
-                        <Item Key="Type"></Item>
-                        <Item Key="Block"></Item>
-                        <Item Key="AccessKey"></Item>
-                        <Item Key="Prio">200</Item>
-                    </Hash>
-                </Item>
```
Comment (dennykorsukewitz) [inline Kernel/Config/Files/XML/Framework.xml:7418]: Can you please undo the change.
The change does not fit thematically with the others (subject). 
Keep it simple. You are welcome to use another pull request.

File: Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt [hunk 3]
```diff
@@ -209,9 +209,6 @@
 
                                 <div class="SettingDescription">
                                     [% Translate(Data.Desc) | html %]
-                                [% IF Data.Label == 'Language' %]
-                                    [% Translate("Did you know? You can help translating OTRS at %s.") | html | ReplacePlaceholders('<a href="https://translate.otrs.com" target="_blank">translate.otrs.com</a>') %]
-                                [% END %]
```
Comment (dennykorsukewitz) [inline Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt:214]: Can you please undo the change.
The change does not fit thematically with the others (subject). 
Keep it simple. You are welcome to use another pull request.

**Review by dennykorsukewitz — CHANGES_REQUESTED** [review]

**Comment by dennykorsukewitz** [comment]
@pboguslawski 

Please solve the conflicting files:
`Kernel/Config/Files/XML/CloudServices.xml`

File: Kernel/Config/Files/XML/Daemon.xml (hunk 1 above)
Comment (pboguslawski) [reply Kernel/Config/Files/XML/Daemon.xml:741]: This task should be enabled only if admin expicity wants it (it wastes resources). We won't remove it. Feel free to adapt to your needs.

File: Kernel/Config/Files/XML/Framework.xml (hunk 2 above)
Comment (pboguslawski) [reply Kernel/Config/Files/XML/Framework.xml:7418]: Subject changed. This ad should be removed and wont be removed from this PR. Feel free to adapt to your needs.

File: Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt (hunk 3 above)
Comment (pboguslawski) [reply Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt:214]: Subject changed. This ad should be removed and wont be removed from this PR. Feel free to adapt to your needs.

**Comment by pboguslawski** [comment]
Conflict in CloudServices.xml resolved.

**Comment by dennykorsukewitz** [comment]
Hi @pboguslawski

I think you have misunderstood me.
If your PR requires multiple labels / descriptions to be applied, you'll most likely need to split it into multiple PRs. 
This makes things easier and faster for the code review. In the future we can better find and understand the changes.

**We will not merge pull requests whose code change includes multiple topics!
Changing the title is not a solution, it is not acceptable!** 👎🏼 


I'm sorry, but if you don't give us a better reason for your code changes and we don't want to adopt the code, then the code stays as it is. 

We are always open and happy to discuss the changes. 👍🏼 


`Feel free to adapt to your needs.` - Paweł We have to think about all users of this system.

File: Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt (hunk 3 above)
Comment (dennykorsukewitz) [reply Kernel/Output/HTML/Templates/Standard/AgentPreferences.tt:214]: **We will not merge pull requests whose code change includes multiple topics!
Changing the title is not a solution, it is not acceptable! **
Please create a new pull request for this.

File: Kernel/Config/Files/XML/Framework.xml (hunk 2 above)
Comment (dennykorsukewitz) [reply Kernel/Config/Files/XML/Framework.xml:7418]: **We will not merge pull requests whose code change includes multiple topics!
Changing the title is not a solution, it is not acceptable! **
Please create a new pull request for this.

File: Kernel/Config/Files/XML/Daemon.xml (hunk 1 above)
Comment (dennykorsukewitz) [reply Kernel/Config/Files/XML/Daemon.xml:741]: If 'ReadOnly' is removed you can adapt to your needs.
If you have no more reason - We will not disable this SysConfig by default.

**Comment by pboguslawski** [comment]
> We will not merge pull requests whose code change includes multiple topics!
> Changing the title is not a solution, it is not acceptable!

Previous upstream stuff cleanup is maintained by us in one branch. Feel free to close this PR it you don't like it or use it as a base for your own cleanups.

**Comment by dennykorsukewitz** [comment]
The PullRequest is almost OK.
1. Two topics in one PullRequest are not clear enough.
   Please simply create another one and everything will be fine.

2. Simply make the change and then we can merge.

**Comment by pboguslawski** [comment]
> Two topics in one PullRequest are not clear enough

This PR turns off (when parameter is available in sysconfig) or removes from code (when function is hardcoded without parameter in sysconfig) previous upstream integration/ad stuff. It is ONE topic. Separating removal of hardcode to different PR costs time - feel free to do it yourself if you still see its sensible (we don't). If you only want us to make another PR from same source branch without other adjustments - this will not be a problem - just let us know.

**Comment by dennykorsukewitz** [comment]
Since our PullRequest rules were not followed and a discussion with the contributor did not bring any further success, we close this PullRequest and solve the problem with the following commit.

https://github.com/znuny/Znuny/commit/d4913719767c08d280d5e8ea4fd6f6ab6c442a26

**Pull request closed without merging**